Source/Extensions/Blazorise.Video/Video.razor.cs

```diff
@@ -481,6 +484,19 @@ public Task NotifyReady()
         return Task.CompletedTask;
     }
 
+    /// <summary>
+    /// Notifies the video component that the quality has changed. Should not be called directly by the user!
+    /// </summary>
+    /// <returns>A task that represents the asynchronous operation.</returns>
+    [JSInvokable]
+    public Task NotifyQualityChange( int? quality )
+    {
+        if ( QualityChanged != null )
+            return QualityChanged.Invoke( quality );
+
+        return Task.CompletedTask;
+    }
```

Comment:
Do you mean `quality` parameter? Well, I think in this context `null` as a valid value, so I would say yes.

Reply:
So if I have a quality set to 1080p and it is invoked as null nothing happens? 
Is there like a default? And if the default is 720p should it trigger with that?

Reply:
It would trigger with anything that `plyr` fires. But since the user picks quality from the dropdown menu on the video player it is highly unlikely it will be null.